src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -68,7 +72,25 @@ define(function (require, exports, module) {
             /**
              * @type {Boolean}
              */
-            disabled: React.PropTypes.bool
+            disabled: React.PropTypes.bool,
+
+            /**
+             * True if the LayerGroup should render collapsed layers. This is controlled by the 
+             * root LayerGroup.
+             * 
+             * @type {Boolean}
+             */
+            renderCollapsedLayers: React.PropTypes.bool
+        },
+        
+        componentDidMount: function () {
+            if (!this.props.parentLayer) {
+                this.renderCollapsedLayers = true;
+
+                window.setTimeout(function () {
+                    this.forceUpdate.bind(this, null);
```

Comment:
Ah, I see what you mean now. I forgot to revet this change when I was experimenting `setTimeout(this.forceUpdate.bind(this, null), 2000)`. Nice catch! I've fixed it and added the comment to clear the confusion.